week03/assignment03_test.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func TestRate(t *testing.T) {
+
+	var tests = []struct {
+		a    Movie
+		want error
+	}{
+		{Movie{Name: "Gladiator", Length: 155, plays: 4}, nil},
+		{Movie{Name: "Léon: The Professional", Length: 110}, fmt.Errorf("can't review a movie without watching it first")},
+	}
+
+	for _, tt := range tests {
+		t.Run("", func(t *testing.T) {
```

Comment:
Always name your subtests. We'll learn why next week. :)

```suggestion
		t.Run(tt.name, func(t *testing.T) {
```

Reply:
ok, cool. I was confused about naming the subtests but it all cleared up :)